src/components/schema/jsonld.tsx

```diff
@@ -0,0 +1,36 @@
+import {ClothingStore, LocalBusiness, Organization, Thing, WithContext, PostalAddress} from 'schema-dts';
+
+export function JsonLd<T extends Thing>(json: WithContext<T>): string {
+  return `<script type="application/ld+json">
+${JSON.stringify(json)}
+</script>`;
+}
+
+export const SchemaWrapper = (data: any) => {
+  return JsonLd<ClothingStore>({
+    '@context': 'https://schema.org',
+    '@type': 'ClothingStore',
+    "@id": data.document.streamOutput._site.uid,
+    name: data.document.streamOutput.name,
+    address: parseAddress(data),
```

Comment:
This would be able to pass and Address type once we define that
```suggestion
    address: parseAddress(data.document.streamOutput.address),
```